Tests/Editor/Actions/StripProBuilderScripts.cs

```diff
@@ -0,0 +1,48 @@
+﻿using UObject = UnityEngine.Object;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEditor.ProBuilder;
+using UnityEngine;
+using UnityEngine.ProBuilder;
+using UnityEditor;
+
+public class StripProBuilderScripts
+{
+    bool m_OpenedWindow = false;
+    ProBuilderMesh m_cube;
+    GameObject m_cube_obj;
+
+    [Test]
+    public void Strip_ProBuilder_Scripts()
```

Comment:
Test names should at a glance indicate what they are testing. In this case I understand that it is validating some behaviour for the script stripping function, but it's unclear exactly what. Consider `StripProBuilderScripts_RemovesMeshAndPolyShapeComponents` or `StripScripts_RemovesAllProBuilderComponents`?

Probuilder isn't the best role model in this regard at the moment :)